thorlcr/activities/filter/thfilterslave.cpp

```diff
@@ -157,6 +157,11 @@ class CFilterSlaveActivity : public CFilterSlaveActivityBase, public CThorSteppa
         CThorSteppable::setInputStream(index, input, consumerOrdered);
     }
     virtual IInputSteppingMeta *querySteppingMeta() { return CThorSteppable::inputStepping; }
+    virtual void getMetaInfo(ThorDataLinkMetaInfo &info) const override
+    {
+        PARENT::getMetaInfo(info);
+        info.fastThrough = true;
```

Comment:
ok, filter isn't "fast through", but for the way it's being used I think filter and project should be.